packages/react-components/src/Concepts/MigratingFromV8.stories.mdx

```diff
@@ -0,0 +1,158 @@
+
+What we are delivering in version 9.x.x-beta are the first fruits of that decision.
+
+Version 9 is a library built-up from the ground-up that addresses many of the concerns and issues that plagued version 8. However, because we have been building components from the ground-up the number of components that we are initially releasing in BETA is much smaller than the amount that existed in version 8. Some of them might also now exist under a different name.
+
+Below is a list of the components that we are releasing with version 9.x.x-beta and what their respective equivalents are in version 8.x.x:
+
+| Component name in v9 | Equivalent component name in v8    |
+| -------------------- | ---------------------------------- |
+| `Accordion`          | No equivalent in v8                |
+| `Avatar`             | `Persona`                          |
+| `Badge`              | No equivalent in v8                |
+| `Button`             | `DefaultButton`                    |
+| `Divider`            | `Separator`                        |
+| `FluentProvider`     | `ThemeProvider`                    |
+| `Link`               | `Link`                             |
+| `Menu`               | `ContextualMenu`                   |
+| `MenuButton`         | `DefaultButton` with `menu` prop   |
+| `Popover`            | `Callout`                          |
+| `Portal`             | `Layer`                            |
+| `Text`               | `Text`                             |
+| `ToggleButton`       | `DefaultButton` with `toggle` prop |
+| `Tooltip`            | `Tooltip`                          |
+
+---
+
+#### "I tried using the new components but there does not seem to be any styling applied to them. Am I doing something wrong?"
+
+Components in version 8 could be used in isolation and still retain their default styling. For version 9, components require to be wrapped by `FluentProvider` and for a theme to be passed to it so that they are styled.
+
+We recommend placing the `FluentProvider` at the root of your app so that everything gets styled appropriately. The theme to use if you are coming from version 8 is `webLightTheme`.
+
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { FluentProvider, webLightTheme } from '@fluentui/react-components';
+
+import App from './App';
+
+ReactDOM.render(
+  <FluentProvider theme={webLightTheme}>
+    <App />
+  </FluentProvider>,
+  document.getElementById('root'),
+);
+```
+
+---
+
+#### "I managed to get the theme working but the components look different than they did previously, why is that?"
+
+We our modernizing our components in version 9 to adhere to the latest guidelines in the Fluent Design Language. That is the reason of any visual differences you might notice.
+
+---
+
+#### "What if I want to override a component's styles? Say, if I want to make them look like they did before? It seems like the styles prop does not exist anymore!"
```

Comment:
I double-checked because I really didn't know which one was the correct usage and it seems like putting it as one word is the correct thing to do here:

https://www.grammarly.com/blog/anymore-vs-any-more/

Reply:
Huh okay, "anymore" still seems weird to me as one word in almost all contexts, but I guess that's just me 🤷‍♀️